ts/screens/wallet/ConfirmSaveCardScreen.tsx

```diff
@@ -95,7 +101,14 @@ class ConfirmSaveCardScreen extends React.Component<Props, State> {
           </Grid>
         </Content>
         <View footer={true}>
-          <Button block={true} primary={true}>
+          <Button
+            block={true}
+            primary={true}
+            onPress={() => {
+              this.props.addCard();
+              this.props.navigation.navigate(ROUTES.WALLET_HOME);
```

Comment:
I'm not sure this makes sense, it seems like I click on the "add" button and then I'm immediately sent to the wallet home - at least I should have some feedback that the card is being saved, and it it fails I should have the opportunity to retry the action without re-entering the card details.